packages/core/database/lib/index.js

```diff
@@ -58,6 +60,13 @@ class Database {
     return schema ? trx.schema.withSchema(schema) : trx.schema;
   }
 
+  async getDatabaseInformation() {
+    if (!this.databaseInformation) {
+      this.databaseInformation = this.dialect.schemaInspector.getDatabaseInformation();
```

Comment:
Shouldn't you store the result of the promise ? (is this a promise ?)
```suggestion
      this.databaseInformation = await this.dialect.schemaInspector.getDatabaseInformation();
```

Reply:
Yes, I forgot the await!